### PR TITLE
feat/DCV-1108 add generate metadata to dbt-coves

### PR DESCRIPTION
**Pull request by BAntonellini**
- Added new command 'generate metadata'
- Moved get_schemas and select_schemas implementation from sources.py to base.py (shared with metadata)
- General linting with black 99 characters

### CODE REVIEW

File: dbt_coves/tasks/generate/metadata.py [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+        )
+
+        cls.arg_parser = base_subparser
+        subparser.set_defaults(cls=cls, which="metadata")
+        return subparser
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.db = None
+        self.METADATA_HEADERS = [
```
Comment (ssassi) [inline dbt_coves/tasks/generate/metadata.py:56]: Define constants at class level, not in `__init__`.

File: dbt_coves/tasks/generate/metadata.py [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+
+        for rel in rels:
+            csv_dest = self.generate_template(destination, rel)
+            csv_path = Path().joinpath(csv_dest)
+            existing_rows = self.get_existing_csv_rows(csv_path)
+            if not options["append_all"]:
+                if csv_path.exists() and existing_rows:
+                    console.print(f"[yellow]{csv_path.absolute()}[/yellow] exists.")
+                    append = questionary.select(
+                        f"Would you like to add {rel.name.lower()} metadata?",
```
Comment (ssassi) [inline dbt_coves/tasks/generate/metadata.py:231]: f"How would you like to append {rel.name.lower()} columns' metadata?"

File: dbt_coves/tasks/generate/metadata.py [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+                        choices=[
+                            "Recreate file",
+                            "Add missing columns",
+                            "Add for all relations",
```
Comment (ssassi) [inline dbt_coves/tasks/generate/metadata.py:236]: This option is confusing.

Comment (BAntonellini) [reply dbt_coves/tasks/generate/metadata.py:236]: @ssassi `add for all relations` is appending to all relations, and not asking again. That's why it toggles `append_all`
should I remove it? And let `Add missing columns` append all csv rows for all relations

another detail I found: when using `Recreate file` shouldn't we presume an `append_all` behavior?

**Pull request merged**